Library/Formula/ice.rb

```diff
@@ -18,6 +30,34 @@ class Ice < Formula
   patch :DATA
 
   def install
+    resource("berkeley-db").stage do
+
+      (Pathname.pwd).install resource("berkeley-db-patch")
+      safe_system "/usr/bin/patch", "-g", "0", "-f", "-d", Pathname.pwd, "-p1", "-i", "berkeley-db-5.3.28.diff"
+
+      # BerkeleyDB dislikes parallel builds
+      ENV.deparallelize
+      # --enable-compat185 is necessary because our build shadows
+      # the system berkeley db 1.x
+      args = %W[
+        --disable-debug
+        --prefix=#{prefix}
+        --mandir=#{man}
```

Comment:
@DomT4 I think your notes and my last push were done at the same time :)
Can you just comment on the last version ? there is no more patch for example and the prefix is set to libexec. It install correctly on my laptop